docs/ui/insight/tools/profiler.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Profiler
+linkTitle: Profiler
+description: Analyze commands in real time
+weight: 10
+
+---
+
+Analyze every command sent to Redis in real time.
```

Comment:
Debug, monitor the performance and security of your Redis or Redis Stack instance using the Profiler tool.
Profiler is a tool based on Redis [MONITOR](https://redis.io/commands/monitor) command that streams back every command processed by the Redis server in real-time.
Profiler can help you:
* debug issues with your database
* diagnose issues and identify problematic commands
* identify performance issues and bottlenecks
* detect suspicious activity such as unauthorized access attempts or unusual commands being sent to the Redis server
* spot bugs in an application both when using Redis or Redis Stack as a database and as a distributed caching system

Because [MONITOR](https://redis.io/commands/monitor) streams back all commands, its use comes at a cost of reducing the throughput of your database, so avoid running Profiler in production databases.

RedisInsight also allows you to download the file with all the logs streamed back to analyze them outside of the application. Enable saving before starting the Profiler to save the logs into a .TXT file.